### PR TITLE
add Wild Apricot source and organization

### PR DESCRIPTION
**Pull request by asirota**

### CODE REVIEW

**Comment by asirota** [comment]
> @googlebot I signed it!

File: sources/wild-apricot.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "WILD_APRICOT",
+  "name": "Wild Apricot API",
```
Comment (pfrisella) [inline sources/wild-apricot.json]: Source name doesn't necessarily have to indicate which API it comes from. 
See info for source name property: https://github.com/googledatastudio/ds-data-registry#add-a-source

So, unless you want to be specific that this source is coming from the API, you can probably just call this Wild Apricot.

File: organizations/wild-apricot-personify.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/wild-apricot-personify.json]: use underscore for spaces in filename.
wild-apricot-personify.json -> wild_apricot_personify.json

File: sources/wild-apricot.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/wild-apricot.json]: use underscore for spaces in filename 
wild-apricot.json -> wild_apricot.json

**Pull request closed without merging**